modules/ROOT/pages/getting-started.adoc

```diff
@@ -49,10 +47,12 @@ Because this project is a Maven project, it contains a `pom.xml` file that conta
 <parent>
   <groupId>org.mule.extensions</groupId>
   <artifactId>mule-modules-parent</artifactId>
-  <version>1.6.3</version>
+  <version>1.9.0</version>
 </parent>
 ----
 
+NOTE: Connectors used in Mule 4.4.x and Mule 4.6.x must use parent POM 1.4.x and 1.6.x respectively.
```

Comment:
This line is incorrect/misleading. What the parent pom defines is the minMuleVersion. That MMV needs to be compatible with whatever the target mule version will be, but there's no need to make then match expicitly. There's also no fundamental change with the currently documented behavior other than using 1.9.0 when Java 17 bytecode is needed